### PR TITLE
ff-ap-timer: Add ap-timer and web-ap-timer from FFHO

### LINKED ISSUE #37: Add ffho-ap-timer and ffho-web-ap-timer
These two packages seem quite valuable and are used across communities.
So it makes sense to add them here.

This would also allow to decrease the list of different module sources in FFMUC, as all used packages are then available in the community-packages repository.

Packages are currently available here:
https://github.com/FreifunkHochstift/ffho-packages/tree/master/ffho-web-ap-timer
https://github.com/FreifunkHochstift/ffho-packages/tree/master/ffho-ap-timer

### PR DESCRIPTION
**Pull request by maurerle**
This makes it possible to add these well-known packages without adding a special package source for these.

It also allows to further develop the package in the future and adjust to eventually needed changes.

The ffac-ap-timer package can turn the client_radio on and off, but does not touch mesh or private wifi configuration.
The ffac-web-ap-timer package makes it possible to configure the schedule in the config mode.

closes #37

### CODE REVIEW

**Comment by grische** [comment]
@maurerle do you want to add this as `ff-` instead of `ffac-` as suggested recently in the chat? Or do you want to maintain this package in the future?

**Review by grische — COMMENTED** [review]
Generally speaking, this version looks almost identical to the one we are already using with a few minor code improvements.

Where did you find that the original code was licensed under BSD 2?

**Comment by maurerle** [comment]
I did split this into three commits:

1. takes the old package
2. renaming to ff- and add license from https://github.com/FreifunkHochstift/ffho-packages/blob/master/LICENSE - and remove old migration
3. adjustments to pass luacheck

This can be merged from my side

File: ff-web-ap-timer/i18n/ffho-web-ap-timer.pot
Comment (grische) [inline ff-web-ap-timer/i18n/ffho-web-ap-timer.pot:1]: I guess you also want to rename this file here

Comment (maurerle) [reply ff-web-ap-timer/i18n/ffho-web-ap-timer.pot:1]: Thank you!

**Comment by maurerle** [comment]
I tested this package on a 7520 and it works good:

image

In the web settings, only the daily option is exposed.

This script changes the uci value on a daily base and writes the changes to flash, then restarts the wifi using `wifi` twice a day too.
This might be considered when using this package, but from the long history of this package I did not hear of devices which were broken due to this.

**Review by grische — APPROVED** [review]
Thanks

**Pull request merged**